.github/workflows/main.yml

```diff
@@ -24,10 +24,10 @@ jobs:
 
     steps:
       - name: Checkout code
-        uses: actions/checkout@master
+        uses: actions/checkout@v2
 
       - name: Use OCaml ${{ matrix.ocaml-version }}
-        uses: avsm/setup-ocaml@v1.0.1
+        uses: avsm/setup-ocaml@v1
```

Comment:
Why the revert to v1?

Reply:
I really don't like the format that the GitHub team popularized, but it's explained here. (I added that recently.)
https://github.com/avsm/setup-ocaml#how-to-specify-the-version

Reply:
Ah ok, thank you for the reference!